docs/developer-guide/02-projects/index.md

```diff
@@ -1 +1,51 @@
 # Projects
+
+The Image Builder team works on a variety of projects across the [entire stack](../stack), from web APIs and frontends, to the nitty gritty assembling of artifacts. The projects here are grouped by their deployment layer, of which there are three.
+
+---
+
+## OSBuild
+
+[OSBuild](./osbuild) is the lowest level in our stack of tools. It takes JSON manifests that describe artifacts and builds those artifacts. `OSBuild` aims to make the building of operating system artifacts precise, declarative, and reproducible. `OSBuild` is written in Python.
+
+### Omnifest Toolkit
+
+[otk](./otk) is a tool to transform files called omnifests to inputs for image build tooling; for example [osbuild](./osbuild) manifests. `otk` aims to make it easier to write definitions for distribution maintainers. `otk` is written in Python.
```

Comment:
Given that this is still WIP I would rephrase it (and maybe move last?). Something like: `A new otk tool is being worked on to transform a high level image definition into an osbuild manifest, more details about the status of the discussion/implementation can be found in <link-to-otk-repo>` or something along those lines.